act-rules-format.bs

```diff
@@ -633,27 +646,28 @@ This section is *non-normative*.
 }
 ```
 
-**Example 3:** Aggregate outcomes to a WCAG success criterion
+**Example 3:** Aggregating based on 'Requirement' (eg. WCAG Success Criteria)
 
 ```javascript
 {
-  "@context": "https://raw.githubusercontent.com/w3c/wcag-act/master/earl-act.json",
+  "@context": "context.json",
   "@type": "Assertion",
+  "assertor": "MyTool",
   "subject": {
     "@type": ["WebPage", "TestSubject"],
     "@value": "https://example.org/page1.html"
   }
-  "test": "wcag21:captions-prerecorded",
+  "test": "WCAG21:captions-prerecorded",
   "result": {
     "outcome": "earl:failed",
     "source": [{
```

Comment:
Instead of having the source be assertions from two different rules, maybe we should have assertions from the same rule, and add a pointer to each showing they target different elements.

Reply:
Done.